### PR TITLE
Remove ansible requirement

### PR DESCRIPTION
**Pull request by mnecas**
Remove ansible requirement for oVirt host el9 support

### CODE REVIEW

File: ovirt-ansible-collection.spec.in [hunk 1]
```diff
@@ -11,11 +11,6 @@ License:	ASL 2.0 and GPLv3+
 BuildArch:	noarch
 Url:		http://www.ovirt.org
 
-%if 0%{?rhel} == 9
-Requires:	ansible-core >= 2.11.0
-%else
-Requires:	ansible >= 2.9.21
-%endif
```
Comment (mwperina) [inline ovirt-ansible-collection.spec.in:18]: Please add:
```
# TODO: Require ansible-core when both engine and hosts are ready for it
# Requires:    ansible-core >= 2.12.0
```

**Review by mwperina — CHANGES_REQUESTED** [review]

File: ovirt-ansible-collection.spec.in (hunk 1 above)
Comment (mnecas) [reply ovirt-ansible-collection.spec.in:18]: Changed the ansible version to 2.11. On el8 will probably not be ansible 2.12 because of python support.

Comment (mwperina) [reply ovirt-ansible-collection.spec.in:18]: Let's try 2.12, I think Python 3.6 and 3.8 could coexist on the same host

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**